### PR TITLE
Adding Xkcd scrapper and Cricket notifications code

### PR DESCRIPTION
**Pull request by Rohithgilla12**
Xkcd.py will allow you to download xkcd comics to give you break in your busy schedule and Cricket notifications code will allow you to get cricket desktop notifications all the time

### CODE REVIEW

**Comment by srbcheema1** [comment]
Thanks for the PR :)
it looks nice to me.
Can you please make some small changes before we merge this code, I am suggesting you some minor changes.
I would be glad to merge this PR :)

File: CricketNotifications.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                n.update(bat_tem+" :"+runs + "/"+wickets + "Overs : "+str(overs),bat1+" :"+r1+" "+bat2+" :"+r2)
+            except:
+                pass
+    old_overs=overs
+    n.show()
+    if(int(wickets)==10):
+        loop_v=1
+    time.sleep(15)
+# print datapath,series_names
+
```
Comment (srbcheema1) [inline CricketNotifications.py]: remove these useless lines

File: xkcd.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+l=[] #For storing random numbers
+def download_img(data,filename): #Function to download images
+    if (os.path.isdir('XKCD')): #Asserts for existence
+        pass
+    else:
+        os.mkdir('XKCD') #If false create a folder
+    op_file=open('XKCD/'+filename,'wb')
+    op_file.write(data) #Download off
+    print "Downloaded",filename
+choice=input("How many random images you want to download?? xD \n")
```
Comment (srbcheema1) [inline xkcd.py]: add two lines space after a function

File: xkcd.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+for i in range(choice):
+    l.append(str(random.randint(1,1933))) #Last comic till date is 1933
```
Comment (srbcheema1) [inline xkcd.py]: leave one line space after for loop

File: xkcd.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+import random
+import os
```
Comment (srbcheema1) [inline xkcd.py:5]: leave one line space after imports

**Comment by Rohithgilla12** [comment]
Yeah, will do
Can you please list the changes so that i can work and get my PR :)

**Review by srbcheema1 — CHANGES_REQUESTED** [review]
Just small changes regarding proper line spacing

**Comment by srbcheema1** [comment]
You may also have a look at my repo https://github.com/srbcheema1/CheemaFy
These days I keep all my useful scripts there in folder srbScripts.
CheemaFy is just an automation project where I keep all my scripts to automate boring stuff.
You may read the README.md to have a basic idea of that.
I would be glad if you contribute in that project too :)

File: CricketNotifications.py [hunk 5]
```diff
@@ -13,21 +15,26 @@
 series_names=[]
 datapath=[]
 for i in temp:
+
```
Comment (srbcheema1) [inline CricketNotifications.py]: not here actually :rofl: 
I meant above `for loop` for loop is treated as a single block of code.
we are to break code into code blocks

**Comment by Rohithgilla12** [comment]
Yeah i would love to contribute to that :D

File: CricketNotifications.py [hunk 6]
```diff
@@ -13,21 +15,26 @@
 series_names=[]
 datapath=[]
```
Comment (srbcheema1) [inline CricketNotifications.py]: it is to be inserted below this.

File: CricketNotifications.py [hunk 7]
```diff
@@ -13,21 +15,26 @@
...
 for i in temp:
+    
     i=str(i)
     i=i.split('datapath')[1]
     i=i.split('"')
     datapath.append(i[1])
     series_names.append(i[13])
 for i in range(len(datapath)):
+
```
Comment (srbcheema1) [inline CricketNotifications.py]: same not here too

File: CricketNotifications.py [hunk 8]
```diff
@@ -13,21 +15,26 @@
...
+    
     i=str(i)
     i=i.split('datapath')[1]
     i=i.split('"')
     datapath.append(i[1])
     series_names.append(i[13])
 for i in range(len(datapath)):
+    
     print str(i+1)+")"+series_names[i]
+
```
Comment (srbcheema1) [inline CricketNotifications.py]: this one is fine :+1:

File: xkcd.py [hunk 9]
```diff
@@ -12,10 +16,15 @@ def download_img(data,filename): #Function to download images
...
     print "Downloaded",filename
+
+
 choice=input("How many random images you want to download?? xD \n")
 for i in range(choice):
+
     l.append(str(random.randint(1,1933))) #Last comic till date is 1933
+
 for i in l:
+
```
Comment (srbcheema1) [inline xkcd.py]: same not here

**Comment by srbcheema1** [comment]
@Rohithgilla12 why did you close the PR ? 
I found it interesting and I was looking forward to merge it

**Comment by Rohithgilla12** [comment]
I am new to this pulling and merging business so I don't have much idea how these stuff goes on so I thought to close and open another issue

**Comment by Rohithgilla12** [comment]
Sorry for that

**Comment by Rohithgilla12** [comment]
I opened another PR please check that

**Comment by srbcheema1** [comment]
ok no problem :)
you will soon have a grip over it.
keep contributing :)

**Comment by Rohithgilla12** [comment]
Thanks dude, can you please merge that PR

**Comment by srbcheema1** [comment]
ya sure.
now i am having lunch. i will surely review and merge it soon :)

**Comment by Rohithgilla12** [comment]
Dude did you review ? #Excited

**Comment by srbcheema1** [comment]
I have reviewed.
Sorry for the delay.
I have my exams going on. So I forgot to open lappi after lunch.
I have made a small review. have a look at it.
just for curiousity, do you know me in real? or met on github only?
and from where are you? and your college?

**Comment by Rohithgilla12** [comment]
Only on github;
I m from Mahindra Ecole Centrale
All the best for your exams :)

**Pull request closed without merging**